frontend/test/unit/specs/components/VMediaTag/v-media-tag.spec.js

```diff
@@ -26,10 +24,9 @@ describe("VMediaTag", () => {
       href: "https://example.com/",
     }
 
-    const wrapper = render(VMediaTag, options)
-    expect(wrapper.vm.$props.tag).toEqual("a")
-    expect(wrapper.vm.$el).toBeInstanceOf(HTMLAnchorElement)
```

Comment:
Is this check no longer worth having?

Reply:
It's fully redundant with checking the tag name. If the tag name is "A" it will be an anchor element unless we don't think the DOM implementation works correctly.